docs/docs/reference/api.md

```diff
@@ -505,9 +505,9 @@ The request parameters associated with this is API are as follows:
 
 ##### Parameters
 
-| Name           | Located in | Description | Required | Type   |
-| -------------- | ---------- | ----------- | -------- | -------- |
-| account_id     | query      |             | No | string   |
```

Comment:
let's keep this `account_id` and add deprecation note

Reply:
done